packages/riipen-ui/src/components/RadioButton.jsx

```diff
@@ -39,10 +39,7 @@ const RadioButton = ({
         size
       )}
     >
-      <label
-        className={clsx(checked && "checked", disabled && "disabled")}
-        htmlFor={other.id || other.name}
```

Comment:
Do we not need htmlFor?

Reply:
this one wrapps the input so the label takes the child input, which is always what we want as its the text on the button